modules/nw-ptp-introduction.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="ptp-introduction_{context}"]
+= Introduction to PTP
+
+The Precision Time Protocol (PTP) is a protocol used to synchronize clocks in a network. When used in conjunction with hardware support, PTP is capable of sub-microsecond accuracy, which is far more accurate than Network Time Protocol (NTP).
+
+The linuxptp package includes the `ptp4l` and `phc2sys` programs for clock synchronization. The `ptp4l` program implements the PTP boundary clock and ordinary clock. The `ptp4I` program synchronizes the PTP hardware clock to the master clock with hardware time stamping and synchronizes the system clock to the master clock with software time stamping. The `phc2sys` program is only needed for hardware time stamping to synchronize the system clock to the PTP hardware clock on the network interface card (NIC).
+
+[id="ptp-elements_{context}"]
+== Elements of a PTP domain
+
+PTP included multiple nodes connected through the network, each representing a clock. The following type of clocks can be included in configurations:
+
+Grandmaster clock:: The grandmaster clock is the reference clock that provides standard time information to other clocks across the network.  The grandmaster clock needs to be able to ensure accurate and stable synchronisation. The grandmaster clock writes time stamps and responds to time requests from other clocks.
+
+Ordinary clock:: The ordinary clock has a single port connection that can play the role of master or slave, depending on its position in the network. The ordinary clock can read and write time stamps.
+
+Boundary clock:: The boundary clock has ports in two or more communication paths and can be a slave and a master of other slaves at the same time. The boundary clock works as a slave upstream: the slave clock receives the timing message, adjusts for delay, and then creates a new master time signal to pass down the network. The boundary clock produces a new timing packet that is still correctly synced with the master, and can reduce the number of connected devices reporting directly to the master. 
```

Comment:
maybe better to switch the 'slave' to `Ordinary` or something else?

Reply:
How about source and destination clocks?